scripts/babel-relay-plugin/src/RelayQLPrinter.js

```diff
@@ -553,6 +554,19 @@ function validateConnectionField(field: RelayQLField): void {
       field.getName()
     );
   });
+
+  if (!connectionNodeType.alwaysImplements('Node')) {
```

Comment:
In a no-longer-surprising-anymore turn of events, that is exactly what i was going to do if people liked the general idea of this PR ;-)